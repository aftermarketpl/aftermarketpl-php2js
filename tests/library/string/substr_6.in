$a = "aBcD";
$b = 2;
$c = 1;
return substr($a, $b, $c);
