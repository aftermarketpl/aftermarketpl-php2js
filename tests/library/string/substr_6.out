a = "aBcD";
b = 2;
c = 1;
return (function(s, x, y){ if(y < 0) return s.substring(x, s.length+y); else return s.substr(x, y); })(a, b, c);
